yarn.lock

```diff
@@ -10340,7 +10340,7 @@ cacheable-request@^6.0.0:
   dependencies:
     clone-response "^1.0.2"
     get-stream "^5.1.0"
-    http-cache-semantics "^4.0.0"
+    http-cache-semantics "^4.1.1"
```

Comment:
How did this change without any update to the `integrity` field?!?

Reply:
I guess manually, but I'm more surprised that yarn isn't complaining (or maybe Yarn is smart enough to not complain if semver allows it)!

Reply:
Yeah I was surprised that yarn didn't complain as well. I would normally expect yarn to do that. But even if yarn doesn't complain, I don't think we should change it like this as there's no need to do so. All that's needed (since this change is within the allowed semver range) is to delete the package entry from the `yarn.lock` file and run the following command:

```sh
yarn cache clean http-cache-semantics $ yarn kbn bootstrap
```

That should _just_ update that section and not any of the dependants.

Reply:
> I don't think we should change it like this as there's no need to do so.

++, we definitely shouldn't do this. Removing the entry (I usually remove all entries to potentially get semver bump for other "semver groups" too) is the only manual change we're supposed to do.